sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -297,7 +297,7 @@ private async Task WaitForEnvironmentInternalAsync()
             }
             else
             {
-                await ExtendResourceGroupExpirationAsync();
+                //await ExtendResourceGroupExpirationAsync();
```

Comment:
Is this intentional?

Reply:
I think this was a temporary workaround for local testing in gov cloud. This would need to be handled in a separate PR to fix in the Test Framework.

Reply:
yes - this will get reverted back before merging :)